fp/api/management/commands/crawl_docs.py

```diff
@@ -0,0 +1,12 @@
+from scrapy.crawler import CrawlerProcess
```

Comment:
Also this one, purpose?

Reply:
These are commands that are executed through django in the shell. I had added them in the past and didn't remove them. They don't use scrapyd btw.

Reply:
Also this one

Reply:
commit 31f613d600c9dbdecaa9a18754b1979d0c2755cf resolves this